flow/instrumentation.cc

```diff
@@ -300,10 +313,6 @@ void CounterValues::Visualize(SkCanvas* canvas, const SkRect& rect) const {
   canvas->drawRect(marker_rect, paint);
 }
 
-int64_t CounterValues::GetCurrentValue() const {
```

Comment:
This seems like a drive by removal of unused code?

Reply:
(which is fine by just making sure I understand and I'm not getting confused by GitHub's diff presentation)

Reply:
@dnfield Yes, it's a removal of unused code.